scripts/scil_ft_visualize_collisions.py

```diff
@@ -0,0 +1,126 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Visualize collisions found through the intersection filtering process of
+fibertubes (see ft_filter_collisions.py). The previously obtained
+"colliders" tractogram is to be used as input in the present script. The
+collision points need to be stored as data_per_streamline on each of its
+streamlines.
+"""
+import argparse
+import nibabel as nib
+
+from dipy.io.streamline import load_tractogram
+from scilpy.io.streamlines import load_tractogram_with_reference
+from fury import window, actor
+from nibabel.streamlines import detect_format, TrkFile
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_verbose_arg,
+                             add_reference_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('colliders',
+                   help='Tractogram file containing the colliding \n'
+                   'streamlines and their collision points (must be .trk). \n')
+
+    p.add_argument('--collided',
+                   help='Tractogram file containing the streamlines that \n'
+                   'have been collided with (must be .trk).')
```

Comment:
Can you say what you are going to do with this trk. It will be overlaid in the viewing window?